guides/source/command_line.md

```diff
@@ -209,7 +209,7 @@ Description:
     Create rails files for model generator.
 ```
 
-NOTE: For a list of available field types, refer to the [API documentation](http://api.rubyonrails.org/classes/ActiveRecord/ConnectionAdapters/TableDefinition.html#method-i-column) for the column method for the `TableDefinition` class.
+NOTE: For a list of available field types, refer to the [API documentation](http://api.rubyonrails.org/classes/ActiveRecord/ConnectionAdapters/SchemaStatements.html#method-i-add_column) for the add_column method for the `SchemaStatements` module.
```

Comment:
Thanks for the explanation! If that's the intent, I think there might be a way to change the wording on the Command Line guides page to explain the greater intent of what's being linked to, but I can't think of what that would be right now, so it's probably good as-is.

It does still feel like two clicks to get to the list of field types is a bit unfortunate, since that's such a common thing to look up—it's confused me more than once as I've tried to find it. It seems like something that could actually be added into the guides themselves, [as in the Laravel framework](https://laravel.com/docs/5.2/migrations#creating-columns). If there's any interest in thinking through ways to make it easier for users to get access to this data, I'm happy to be in the convo and do the doc updates for it!